### PR TITLE
Extend the Polaris helm chart to support an 'ingressClassName'

### LINKED ISSUE #589: Polaris - Support IngressClassName
**Is your feature request related to a problem? Please describe.**
For Ingress, the `kubernetes.io/ingress.class` annotation is deprecated.
Instead, an `ingressClassName` should be used in Ingress objects.

**Describe the solution you'd like**
Add support for a `dashboard.ingress.ingressClassName` property in the Helm chart to be able to set the `ingressClassName` in the Ingress object.

**Additional context**
Ideally, this should be applied to all charts.

### PR DESCRIPTION
**Pull request by diversit**
**Why This PR?**
The ingress annotation `kubernetes.io/ingress.class` is deprecated and replaced by an `ingressClassName` property in the IngressSpec.

Fixes #589 

**Changes**
Changes proposed in this pull request:

* Support property `dashboard.ingress.ingressClassName` when Kubernetes version supports it.

**Checklist:**

* [x] I have updated the chart version in `Chart.yaml` following Semantic Versioning.
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] Any new values have been added to the README for the Chart, or helm-docs has been run for the charts that support it.

### CODE REVIEW

File: stable/polaris/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,11 @@ All notable changes to this Helm chart will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this chart adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## 4.2.1
+
+### Added
+Support `ingressClassName` in Ingress object.
```
Comment (rbren) [inline stable/polaris/CHANGELOG.md:11]: clearly we haven't been very consistent here 😄

**Review by rbren — APPROVED** [review]
Thanks for the contribution!

**Comment by paul-ktech** [comment]
Hi, can we get this one merged in ready for next release? Thanks!

File: stable/polaris/templates/ingress.yaml [hunk 2]
```diff
@@ -17,6 +17,9 @@ metadata:
     app.kubernetes.io/instance: {{ .Release.Name }}
   name: polaris
 spec:
+{{- if and (.Values.dashboard.ingress.ingressClassName) (semverCompare ">= v1.18.0" .Capabilities.KubeVersion.Version) }}
```
Comment (fredgate) [inline stable/polaris/templates/ingress.yaml]: It is better to stay consistent and use the same condition as at the top of the file to choose the API version:
```suggestion
{{- if and (.Values.dashboard.ingress.ingressClassName) (.Capabilities.APIVersions.Has "networking.k8s.io/v1/Ingress") }}
```

File: stable/polaris/Chart.yaml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Validation of best practices in your Kubernetes clusters
 name: polaris
-version: 4.2.0
+version: 4.2.1
```
Comment (fredgate) [inline stable/polaris/Chart.yaml]: The chart `4.2.1` is already released.
```suggestion
version: 4.2.2
```

**Pull request merged**